packages/cms-lib/fileManager.js

```diff
@@ -174,6 +154,19 @@ async function fetchAllPagedFiles(accountId, folderId, { includeArchived }) {
  * @param {object} options
  */
 async function fetchFolderContents(accountId, folder, dest, options) {
+  try {
+    await fs.ensureDir(dest);
+    logger.log('Wrote folder "%s"', dest);
```

Comment:
Minor, but I think that this assumes that the folder needs to be created.

Reply:
Good point, I just removed for now since I directory creation is implied